front/src/applications/operationalStudies/views/Scenario/components/MacroEditor/ngeToOsrd.ts

```diff
@@ -983,12 +990,24 @@ const handleNodeOperation = async ({
         if (indexNode.dbId) {
           // Update the key if trigram has changed and key is based on it
           let nodeKey = indexNode.path_item_key;
-          if (nodeKey.startsWith('trigram:') && indexNode.trigram !== node.betriebspunktName) {
-            nodeKey = `trigram:${node.betriebspunktName}`;
+          let trigram = node.betriebspunktName;
+          if (nodeKey.startsWith('trigram:') && indexNode.trigram !== trigram) {
+            if (trigram.includes('/')) {
```

Comment:
I think this condition should be inverted?

Reply:
Done